content/blog/2016/2016-01-10-beautiful-jenkins-dashboard.adoc

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: "A beautiful Jenkins dashboard"
+author: kj187
+---
+
+
+image:http://res.cloudinary.com/kj187/image/upload/c_scale,w_890/v1450212394/kj187_Dashboard_2_i4wkw6.png[Dashboard, 250, float="right", link="goo.gl/X3WM3r"]
+
+Jenkins offers a handful of third party dashboards, but none of them is really beautiful and flexible from my point of view.
+
+So I decided to write a new, flexible and extensible dashboard. To avoid re-inventing the wheel I decided to use link:https://github.com/fabiocaseri/dashing-js[Dashing-JS] as a basis and not Jenkins itself. Dashing-JS is a NodeJS port of link:http://dashing.io/[Dashing], an Sinatra based framework that lets you build beautiful dashboards. This is completely independent of Jenkins. All you need is NodeJS and the permission to access the Jenkins API.
```

Comment:
Perhaps more explanation of what Dashing is, or perhaps why it's better to do this external to Jenkins?